Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -2301,12 +2301,22 @@ theorem le_centralizer_iff : H ≤ K.centralizer ↔ K ≤ H.centralizer :=
 #align subgroup.le_centralizer_iff Subgroup.le_centralizer_iff
 #align add_subgroup.le_centralizer_iff AddSubgroup.le_centralizer_iff
 
+@[to_additive]
+theorem center_le_centralizer (s) : center G ≤ centralizer s :=
+  Set.center_subset_centralizer (s : Set G)
+#align subgroup.center_le_centralizer Subgroup.center_le_centralizer
```

Comment:
Are you going to add +1 `#align` while forward-porting leanprover-community/mathlib#19168?

Reply:
Yes, when it merges in I'll sort the #aligns, I think it would currently error as there would be no declarations of the right name.

Reply:
It won't error, #align doesn't check if the Lean3 name exists